frontend/src/metabase/dashboard/reducers.unit.spec.js

```diff
@@ -42,6 +42,13 @@ describe("dashboard reducers", () => {
         toastId: null,
         toastDashboardId: null,
       },
+      autoWireParameters: {
```

Comment:
There was a similar issue with one of the PRs @npretto worked on where there was a piece of state for toasts when it wasn't needed. In fact, this change is a burden to maintain. Could you discuss this and avoid modifying the global state?

Reply:
https://github.com/metabase/metabase/blob/cc50e7a94216ae738cb4aad609122abcf8082e32/frontend/src/metabase/dashboard/actions/tabs.ts#L102-L131
This is the solution I came up with, instead of storing the values I needed in redux, I basically stored them in the closure of the undo

Reply:
Gotcha, I think a combo of Nicolo's and Kelvin's solution will do me good here